client/my-sites/sharing/buttons/options.jsx

```diff
@@ -211,10 +215,15 @@ const connectComponent = connect(
 		const siteId = getSelectedSiteId( state );
 		const isJetpack = isJetpackSite( state, siteId );
 		const isTwitterButtonAllowed = ! isJetpack || isJetpackMinimumVersion( state, siteId, '3.4-dev' );
+		const postTypes = values( getPostTypes( state, siteId ) );
 
 		return {
+			buttons: getSharingButtons( state, siteId ),
```

Comment:
I was debating that as well. I came to the conclusion that this component is very much tied to its parent and that the query components could be added at a point when the component would be used outside of its current context